Tests/Functional/WebTestCase.php

```diff
@@ -20,9 +20,9 @@
 
 namespace FOS\ElasticaBundle\Tests\Functional;
 
-use Symfony\Bundle\FrameworkBundle\Tests\Functional\WebTestCase as BaseWebTestCase;
```

Comment:
this was using the wrong namespace (Tests vs Test) and also no need to use WebTestCase if you just need the container